.github/CODEOWNERS

```diff
@@ -1,3 +1,3 @@
 * @pomerium/dev-backend
-/docs/  @pomerium/docs
+/docs/  @ghost
```

Comment:
Is this valid? Github doesn't seem to like it.

Reply:
It's used elsewhere: https://github.com/coq/coq/blob/master/.github/CODEOWNERS#L52-L55